vhost-device-i2c/src/main.rs

```diff
@@ -306,6 +338,15 @@ mod tests {
     fn test_parse_failure() {
         let socket_name = "vi2c.sock";
 
+        // Space in filenames
+        let cmd_args = I2cArgs::from_args(" ./vi2c.sock", " 1:4", 1);
+        // " ./vi2c.sock" will fail because " ." does not exist, while "." exists in every UNIX
+        // directory.
+        assert_matches!(
+            I2cConfiguration::try_from(cmd_args).unwrap_err(),
+            Error::PathParseFailure(p, msg) if p == Path::new(" ./vi2c.sock") && msg.starts_with("Parent directory `") && msg.ends_with("` does not exist.")
```

Comment:
Maybe create a seprate string for the path:

let path = " ./vi2c.sock"

So we don't need to duplicate the tricky path name.

Reply:
Thanks, fixed